spec/mixlib/shellout/windows_spec.rb

```diff
@@ -307,4 +307,88 @@ def self.with_command(cmd, filename: nil, search: false, directory: false, &exam
       end
     end
   end
+
+  context "#combine_args" do
+    let(:shell_out) { Mixlib::ShellOut.new }
+    subject { shell_out.send(:combine_args, *largs) }
+
+    def self.with_args(*args, &example)
+      context "with command #{args}" do
+        let(:largs) { args }
+        it(&example)
+      end
+    end
+
+    with_args("echo", "%PATH%") do
+      is_expected.to eql(%q{echo %PATH%})
+    end
+
+    with_args("echo %PATH%") do
+      is_expected.to eql(%q{echo %PATH%})
+    end
+
+    # Note carefully for the following that single quotes in ruby support '\\' as an escape sequence for a single
+    # literal backslash.  It is not mandatory to always use this since '\a' does not esacpe the 's' and is literally
```

Comment:
```diff
- esacpe the 's'
+ escape the 'a'
```

Reply:
"a" is probably a bad example, as `"\a"` (double) really is escaped:

```
>> "\d"
=> "d"
>> "\a"
=> "\a"
```

Reply:
heh, i picked '\s' first which was obviously much worse...